### ISSUE TITLE
Travis builds on master failing with "require: cannot load such file -- mixlib/shellout"

### ISSUE THREAD

**Issue by edmorley** [chef/bento#839]
For example:
```
$ bento normalize
/home/travis/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- mixlib/shellout (LoadError)
    from /home/travis/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /home/travis/.rvm/gems/ruby-2.3.1/gems/bento-ya-0.1.0/lib/bento/build.rb:1:in `<top (required)>'
    from /home/travis/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /home/travis/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /home/travis/.rvm/gems/ruby-2.3.1/gems/bento-ya-0.1.0/lib/bento/cli.rb:5:in `<top (required)>'
    from /home/travis/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /home/travis/.rvm/rubies/ruby-2.3.1/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
    from /home/travis/.rvm/gems/ruby-2.3.1/gems/bento-ya-0.1.0/bin/bento:10:in `<top (required)>'
    from /home/travis/.rvm/gems/ruby-2.3.1/bin/bento:23:in `load'
    from /home/travis/.rvm/gems/ruby-2.3.1/bin/bento:23:in `<main>'
    from /home/travis/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `eval'
    from /home/travis/.rvm/gems/ruby-2.3.1/bin/ruby_executable_hooks:15:in `<main>'
```

First failing master build:
https://travis-ci.org/chef/bento/builds/249103739

Last passing master build:
https://travis-ci.org/chef/bento/builds/248858324

**Comment by edmorley** [comment]
The initial breakage has since been fixed by cheeseplus/bento-ya#15 (released in bento-ya 0.1.1), however after that the failure mode changed to:
```
$ bento normalize
/home/travis/.rvm/rubies/ruby-2.4.1/lib/ruby/site_ruby/2.4.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- kitchen (LoadError)
...
```
(See: https://travis-ci.org/chef/bento/builds/249850228)

**Comment by cheeseplus** [comment]
Resolved over in bento-ya, though there are bizarre issues with hyperv-iso throwing issues even though it doesn't locally.
